ros2node/ros2node/verb/info.py

```diff
@@ -38,28 +38,34 @@ def add_arguments(self, parser, cli_name):
             'node_name',
             help='Node name to request information')
         argument.completer = NodeNameCompleter()
+        parser.add_argument(
+            '--include-hidden', action='store_true',
+            help='Display hidden topics, services, and actions as well')
```

Comment:
Maybe a user wants to see hidden nodes but not hidden topics / services / actions - or the other way around? Just a thought...

Reply:
I presume you mean them wanting to see hidden topics, but not hidden services / actions (or some combination). I thought about it, but I figured it's a rare case to want to see any of the hidden names, let alone a subset.